src/pages/patientView/copyNumberAlterations/CopyNumberTableWrapper.tsx

```diff
@@ -57,6 +58,12 @@ export default class CopyNumberTableWrapper extends React.Component<ICopyNumberT
         enableCivic: false
     };
 
+    getCytobandForGene(hugoGeneSymbol:string) {
+        const refGene =  _.find(this.props.referenceGenes,(gene:ReferenceGenomeGene)=>
```

Comment:
this is a linear search through props.referenceGenes every time you want to call this method, which is inefficient and could become a bit slow for a query with a lot of rows and a lot of genes.

Instead,

```
@computed get hugoGeneSymbolToCytoband() {
    if (this.props.referenceGenes) {
        return this.props.referenceGenes.reduce((map, next)=>{
            map[next.hugoGeneSymbol] = next.cytoband;
        });
    } else {
        return {};
    }
}
```